ckanext/iati/views/publisher.py

```diff
@@ -215,6 +219,16 @@ def post(self, group_type, is_organization):
         context['message'] = data_dict.get('log_message', '')
         data_dict['type'] = 'organization'
 
+        captcha_response = data_dict['h-captcha-response']
+        response = requests.post('https://hcaptcha.com/siteverify', data={
+            'secret': os.environ.get('hCAPTCHA_SECRET_KEY'),
+            'response': captcha_response,
```

Comment:
Add a timeout parameter here -- requests should never be used in the latency path without it. Also, probably worth trapping any requests exception here. Not sure if we want to fail open or not.